### PR TITLE
Add browserslist-config-nebenan package

### PR DESCRIPTION
**Pull request by peterhass**

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -1,10 +1 @@
...
-
-last 2 major versions
-last 2 years
-> 1%
-not dead
-not op_mini all
-not op_mob > 0
-not ie > 0
-not ie_mob > 0
+extends @goodhood/browserslist-config-nebenan
```
Comment (suprMax) [inline .browserslistrc:1]: maybe publish as just `browserslist-config-nebenan` and move in a separate repo? this repo holds react ui packages

File: packages/browserslist-config-nebenan/LICENSE [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Copyright (c) Good Hood GmbH.
```
Comment (suprMax) [inline packages/browserslist-config-nebenan/LICENSE:1]: permissive MIT license would work here

File: .browserslistrc (hunk 1 above)
Comment (peterhass) [reply .browserslistrc:1]: Doesn't really matter imo. Doesn't complicate setup in any way

if you only want to have react stuff in here, then you chose a very generic scope name

**Comment by peterhass** [comment]
Decision: Move to own repo

**Pull request closed without merging**